lib/gtfsplus/util/index.js

```diff
@@ -0,0 +1,11 @@
+import { getConfigProperty } from '../../common/util/config'
```

Comment:
Please add flow typing to at least this file.  All other util files are flow-typed, so let's keep that going.